app/scripts/services/logLinks.js

```diff
@@ -65,12 +65,12 @@ angular.module('openshiftConsole')
         ")",
         "&_a=(",
           //"columns:!(_source),",
-          "columns:!(kubernetes_container_name,<%= containername %>),", 
-          "index:'<%= namespace %>.*',",
+          "columns:!(kubernetes_container_name,message),",
```

Comment:
@ewolinetz 's comment in the bugzilla says this should be log, what is the different between log and message here?